#!/usr/bin/python3
def raise_exception():
  x = 1
  y = "2"
  z = x + y
